### DISCUSSION TITLE
有一些自定义的配置文件，放到file目录下后，然后怎么在打包的时候带进去呢？

### DISCUSSION THREAD

**Discussion by wanmyj** [chenmozhijin/OpenWrt-K#54] [Q&A] [answered]
看说明，files应该是存放openwrt固件自定义文件，我有一个nginx的配置文件，需要放在`/etc/nginx/conf.d/`目录下，请问大佬这个要怎么配置？

**Comment by chenmozhijin** [comment, marked as answer]
`files/`文件夹就相当于openwrt中的根目录`/`，所以应该是放到`files/etc/nginx/conf.d/`目录下。

不过需要注意配置文件可能在首次开机时被[UCI defaults](https://openwrt.org/docs/guide-developer/uci-defaults)脚本修改

**Reply by wanmyj** [reply]
谢谢沉默！
